cibyl/sources/jenkins_job_builder.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, url: str, dest: str = None, branch: str = None):
         :param branch: Branch to checkout
         :type branch: str
         """
-        super().__init__("", url)
+        super().__init__("", url=url, driver="jenkins_job_builder")
```

Comment:
We should add the driver attribute to the rest of sources, but we can do that in another PR

Reply:
Yes, but right now only jjb and jenkins I think inherit from Source

Reply:
Yes, I think that's true, we can leave it for a  later PR